packages/functional-tests/tests/misc.spec.ts

```diff
@@ -4,32 +4,6 @@
 
 import { expect, test } from '../lib/fixtures/standard';
 
-test.describe('severity-1', () => {
-  // runs all mocha tests - see output here: http://127.0.0.1:3030/tests/index.html
-  test('content-server mocha tests', async ({ target, page }, { project }) => {
-    test.skip(project.name !== 'local', 'mocha tests are local only');
-    await page.goto(`${target.contentServerUrl}/tests/index.html`, {
-      waitUntil: 'load',
-    });
-    await page.waitForTimeout(2000); // wait for mocha to load
-    await page.evaluate(() =>
-      (globalThis as any).runner.on(
-        'end',
-        () => ((globalThis as any).done = true)
-      )
-    );
-    await page.waitForFunction(
-      () => (globalThis as any).done,
-      {},
-      { timeout: 0 }
-    );
-    const failures = await page.evaluate(
-      () => (globalThis as any).runner.failures
-    );
-    expect(failures).toBe(0);
-  });
-});
-
```

Comment:
If I'm reading right, this was a test to make sure all content-server intern tests completed successfully, and it's safe to remove?